spec/models/has_struct_metadata_spec.rb

```diff
@@ -0,0 +1,33 @@
+require 'spec_helper'
+
+module Ddr
+  module Models
+    RSpec.describe HasStructMetadata, type: :model do
+
+      context "component" do
+        describe "component creation" do
+          let(:item) { Item.new }
+          before do
+            allow_any_instance_of(Component).to receive(:parent).and_return(item)
+            allow_any_instance_of(Component).to receive(:has_content?).and_return(true)
+            allow_any_instance_of(Component).to receive(:content_type).and_return("image/tiff")
```

Comment:
You shouldn't need `allow_any_instance_of` since you are explicitly creating the instance.